docs/source/index.rst

```diff
@@ -91,69 +89,20 @@ The :samp:`init_json` interface is similar to the :ref:`init-yaml` interface
 except that :samp:`spec_filename` must be a JSON file and :samp:`PyYAML` is not
 a required dependency.
 
-.. _init-model-factory:
-
-:samp:`init_model_factory`
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The :samp:`init_model_factory` interface is less user friendly but perhaps of
-interest to advanced users. It accepts the specification in dictionary format
-(so it has fewer dependencies than :ref:`init-yaml` and :ref:`init-json`) and
-does not construct a declarative base. It accepts the following parameters:
-
-* :samp:`base`: The SQLAlchemy declarative base as a keyword only argument. It
-  is used to as the base class for all SQLAlchemy models.
-* :samp:`spec`: The OpenAPI specification as a dictionary as a keyword only
-  argument.
-* :samp:`models_filename`: The name of the file where the SQLAlchemy models
-  will be written as an optional keyword only argument.
-
-.. note:: the :samp:`define_all` parameter has been removed and OpenAlchemy
-  behaves as though it is set to :samp:`True`.
-
-The return value is the :samp:`model_factory` as defined as part of the return
-value of :ref:`init-yaml`.
-
-.. _build-yaml:
-
 :samp:`build_yaml`
 ^^^^^^^^^^^^^^^^^^
 
 Used to build a package with the SQLAlchemy models (including type hints) based
 on a YAML OpenAPI specification which has been extended with any relevant
 OpenAlchemy extension properties.
```

Comment:
It would be good to include a link to read more here, also the case for the other sections where there is a short introduction